_goldens/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ dev_dependencies:
   angular: '^4.0.0'
   angular_forms: '^1.0.0'
   _goldens.generator: ^1.0.0
-  build_runner: ^0.5.0
+  build_runner: '0.4.0+2'
```

Comment:
This feels – weird. By design?

Reply:
Yes, see https://github.com/dart-lang/build/issues/496

Reply:
Add a todo in there to track?